docs/book/installation/sylius_plus_installation.rst

```diff
@@ -136,12 +136,12 @@ Installing Sylius Plus as a plugin to a Sylius application
      * @ORM\Entity
      * @ORM\Table(name="sylius_admin_user")
      */
-    class AdminUser extends BaseAdminUser implements AdminUserInterface, RbacAdminUserInterface, ChannelAwareInterface, LastLoginIpAwareInterface
+    class AdminUser extends BaseAdminUser implements AdminUserInterface, RbacAdminUserInterface, LastLoginIpAwareInterface
```

Comment:
> Class cannot implement previously implemented interface

Because `ChannelAwareInterface` is already in `RbacAdminUserInterface`.

Reply:
The usage of this interface should be removed in line 125.